community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -0,0 +1,4106 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    if PROJECT_ID == "" or PROJECT_ID is None or PROJECT_ID == "[your-project-id]":
</pre><p>if not PROJECT_ID or PROJET_ID == "[...]":</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Done.